Godeps/Godeps.json

```diff
@@ -2791,6 +2791,58 @@
 			"Rev": "c2ac40f1adf8c42a79badddb2a2acd673cae3bcb"
 		},
 		{
+			"ImportPath": "k8s.io/metrics/pkg/apis/custom_metrics",
```

Comment:
Adding k8s.io/metrics as a vendored package was not a good idea, leading to circular repo dependencies: it depends on k8s.io/apimachinery which is in staging/ here. Changes in apimachinery easily trigger compile errors in vendor/k8s.io/metrics. It's more than tedious to fix them by updating k8s.io/metrics first with a new apimachinery and then re-vendor it here.